frontend/survey-project/src/app/components/pages/register-page/register-page.component.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sources":["register-page.component.scss","register-page.component.css"],"names":[],"mappings":"AAAA;EACI,aAAA;EACA,mBAAA;EACA,uBAAA;EACA,iBAAA;ACCJ;;ADEE;EACE,UAAA;EACA,iBAAA;EACA,iBAAA;EACA,kBAAA;EACA,sBAAA;EACA,sCAAA;EACA,mBAAA;EACA,yCAAA;ACCJ;;ADEE;EACE,eAAA;EACA,WAAA;EACA,mBAAA;ACCJ;;ADEE;EACE,iBAAA;EACA,yBAAA;EACA,qBAAA;ACCJ;;ADEE;EACE,gBAAA;ACCJ;;ADEE;EACE,YAAA;EACA,0BAAA;EACA,cAAA;ACCJ;;ADEE;EAEE,cAAA;ACCJ;;ADHE;;EAEE,cAAA;ACCJ","file":"register-page.component.css"}
```

Comment:
Not sure what is this used for, but I think it's not needed, we're using scss not css and the Angular builder will build our scss to css.